src/solidity-frontend/solidity_convert.cpp

```diff
@@ -3415,7 +3597,7 @@ bool solidity_convertert::get_elementary_type_name(
   }
   case SolidityGrammar::ElementaryTypeNameT::BYTES:
   {
-    new_type = uint_type();
+    new_type = unsignedbv_typet(256);
```

Comment:
Why 256?

Reply:
In solidity, Bytes = Bytes32. And we further convert the bytes32 to uint256 in the front end.